addons/mail/models/mail_thread.py

```diff
@@ -3609,6 +3609,11 @@ def _notify_thread_by_web_push(self, message, recipients_data, msg_vals=False, *
         if not partner_ids:
             return
 
+        # web_push in discuss is handled on the client side, see `notifyMessageToUser()`
+        message_model = msg_vals.get('model') if msg_vals else message.model
+        if message_model == 'discuss.channel':
+            return
```

Comment:
This doesn't make a lot of sense.
Aren't web push meant for native notification? Which by definitions can't be handled if client is not opened?